src/pages/index.astro

```diff
@@ -58,6 +59,7 @@ import Footer from '../components/Footer.astro';
       </div>
     </div>
   </main>
+  <Gallery />
```

Comment:
Gallery should be inside main. Placeholder will be exported to separate component later

Reply:
Other than that great work